pkg/component/component.go

```diff
@@ -237,7 +237,7 @@ func PushLocal(client *occlient.Client, componentName string, applicationName st
 		scanner := bufio.NewScanner(pipeReader)
 		for scanner.Scan() {
 			line := scanner.Text()
-			yellowFprintln(out, line)
+			yellowFprintln(color.Output, line)
```

Comment:
:-1: you need to add `color.Output` higher up, we refactored this a while ago and we now pass in `io.Writer` as a parameter.

Have a look at `cmd/create.go` and `cmd/push.go`.

Instead of modifying this line here, you will need to modify it similar to how it's implemented here:

https://github.com/redhat-developer/odo/blob/master/cmd/create.go#L64

By supplying stdout at the beginning and them passing in stdout to the parameter.